internal/core/note_find.go

```diff
@@ -5,14 +5,12 @@ import (
 	"strings"
 	"time"
 	"unicode/utf8"
-
-	"github.com/mickael-menu/zk/internal/util/opt"
 )
 
 // NoteFindOpts holds a set of filtering options used to find notes.
 type NoteFindOpts struct {
 	// Filter used to match the notes with the given MatchStrategy.
-	Match opt.String
+	Match []string
```

Comment:
Dropped the opt string type here and in filter since the collection operates in a similar enough manner (len = 0 == nullString), did not see reason for double encapsulation.

Reply:
Absolutely, that's the right call.